### DISCUSSION TITLE
Over a year since GEDCOM7 spec was published

### DISCUSSION THREAD

**Discussion by emyoulation** [FamilySearch/GEDCOM#173] [Q&A] [answered]
Is there anything which supports the new format? For import, export or validation?

Is it on anyone's announced roadmap?

**Comment by tychonievich** [comment]
https://gedcom.io/tools/ has some products that have been announced and given to us to share (mostly validators and converters so far).

I am aware of many companies that have told members of the GEDOCM steering committee that they have version 7.0 support on their internal developers' roadmap, and some that have made great progress towards that with pre-release implementations they mention in communications with us. However, I do not monitor these company's public announcements and do not know if they are on their announced roadmaps or only their internal roadmaps.

**Comment by ghost** [comment, marked as answer]
Don't forget this list, Luther: https://www.familysearch.org/en/GEDCOM/implementation-progress

Tony
